model/client4.go

```diff
@@ -1795,7 +1795,19 @@ func (c *Client4) PermanentDeleteTeam(teamId string) (bool, *Response) {
 }
 
 // GetTeamMembers returns team members based on the provided team id string.
-func (c *Client4) GetTeamMembers(teamId string, page int, perPage int, sort string, exclude_deleted_users bool, etag string) ([]*TeamMember, *Response) {
+func (c *Client4) GetTeamMembers(teamId string, page int, perPage int, etag string) ([]*TeamMember, *Response) {
+	query := fmt.Sprintf("?page=%v&per_page=%v", page, perPage)
+	r, err := c.DoApiGet(c.GetTeamMembersRoute(teamId)+query, etag)
+	if err != nil {
+		return nil, BuildErrorResponse(r, err)
+	}
+	defer closeBody(r)
+	return TeamMembersFromJson(r.Body), BuildResponse(r)
+}
+
+// GetTeamMembersWithoutDeletedUsers returns team members based on the provided team id string. Additional parameters of sort and exclude_deleted_users accepted as well
+// Could not add it to above function due to it be a breaking change.
+func (c *Client4) GetTeamMembersWithoutDeletedUsers(teamId string, page int, perPage int, sort string, exclude_deleted_users bool, etag string) ([]*TeamMember, *Response) {
```

Comment:
Oops, didn't "see" the `sort` parameter here as well, so my naming suggestion isn't ideal. Should we just name this `GetTeamMebersWithOptions` and accept a `TeamMembersGetOptions`? I guess `ViewRestrictions` doesn't make sense as a parameter to accept here, though.

Reply:
@lieut-data 
Ya exactly, it doesn't make sense cause it may become confusing. I just renamed it to `GetTeamMembersSortAndWithoutDeletedUsers`.